modules/apps/frontend-js/frontend-js-state-web/src/main/resources/META-INF/resources/State.ts

```diff
@@ -503,22 +503,10 @@ function isAtom(value: unknown): value is Atom<any> {
 	return Object.hasOwnProperty.call(value, ATOM);
 }
 
-window.Liferay = window.Liferay || {};
-
-window.Liferay.State = State;
+// @ts-ignore
```

Comment:
Why do we need these ignores in this file?

Reply:
I could've sworn it was complaining that `Liferay` doesn't exist on the `window` object 🤔